etc/org.opencastproject.organization-mh_default_org.cfg

```diff
@@ -52,26 +52,17 @@ name=Opencast Project
 #
 # Example Configuration:
 #
-# prop.org.opencastproject.host.admin-presentation.opencast.com=tenant1.admin-presentation.opencast.com
-# prop.org.opencastproject.host.ingest.opencast.com=tenant1.ingest.opencast.com
-# prop.org.opencastproject.host.worker.opencast.com=tenant1.worker.opencast.com
+# prop.org.opencastproject.host.admin-presentation.opencast.com=http://tenant1.admin-presentation.opencast.com
+# prop.org.opencastproject.host.ingest.opencast.com=https://tenant1.ingest.opencast.com
```

Comment:
Does this mean that you cannot have a tenant that reacts to both http and https any longer?

Reply:
No, if I understand the code correctly.

The schema only matters for the `ServiceRegistryEndpoint` `/available.{xml|json}`. pyCA, for instance, should then only use the schema and port specified here. Note that the other service registry endpoints will still report the URL of the service registration (i.e. `org.opencastproject.server.url`). I would expect them to be tenant-specific as well, but I don't know if this is intentionally limited to this one endpoint.

Opencast currently determines the tenant using only the hostname. You should be able to route traffic through different ports or a different protocol schema. As mentioned in the updated docs, this also means that it is currently not possible to have `tennen1` use `http://localhost:8080` and `tennen2` `http://localhost:8081`.

So including a schema here is necessary for the `ServiceRegistryEndpoint` and to not explicitly specifying a port number.

Reply:
mmh I was wrong. The port is used to differentiate between tenants. Will update the docs.